frontend/ui/widget/inputdialog.lua

```diff
@@ -148,6 +149,23 @@ function InputDialog:init()
         parent = self,
     }
 
+    if self.show_toggle then
+        local button_switch = {
+            {
+                text = ("Switch visible"),
+                callback = function()
+                    if self._input_widget.text_type == "text" then
+                        self._input_widget.text_type = "password"
+                        self:changeTextType("password")
+                    else
+                        self._input_widget.text_type = "text"
```

Comment:
More specifically, I propose to change `.text` like this

`☐ Show password` (hidden)
`√ Show password` (shown)

(Alternatively you could change it to `Show password` but I think that might be a touch disorienting)